ceph.spec.in

```diff
@@ -323,6 +324,8 @@ Requires:	python-flask
 %if 0%{?suse_version}
 # python-Flask for powerdns
 Requires:      python-Flask
+# libopenssl for civetweb https support
+Requires:      libopenssl1_0_0
```

Comment:
Once the libopenssl1 headers and libraries are present during build time (via BuildRequires), the build should produce binaries which link against the libopenssl shared libraries - you can check this via ldd <rgw binary>.
Packaged binaries are checked for shared library dependencies, and these are added as rpm requirements automatically, so an explicit runtime Requires entry shouldn't be necessary, unless the binary is doing something tricky like using dlopen().